docs/root/start/building.rst

```diff
@@ -26,12 +26,31 @@ Building Envoy has the following requirements:
   :ref:`this FAQ entry <faq_build_speed>` for more information on build performance.
 * These :repo:`Bazel native <bazel/repository_locations.bzl>` dependencies.
 
-Please see the linked :repo:`CI <ci/README.md>` and :repo:`Bazel <bazel/README.md>` documentation
-for more information on performing manual builds.
 Please note that for Clang/LLVM 8 and lower, Envoy may need to be built with `--define tcmalloc=gperftools`
 as the new tcmalloc code is not guaranteed to compile with lower versions of Clang.
 
 
+Windows Requirements
```

Comment:
and this is building for a windows target - but are not the requirements if you are building *on* windows for a unix target

in wondering if we can change the titles to something line "Requirements for building a mac/linux/windows binary"